assets/js/main.js

```diff
@@ -52,3 +52,20 @@ $("#btn-modal-small").click(function() {
 $(".modal .close").click(function() {
   $(".modal").fadeOut();
 });
+
+
+// script to show and hide password
+$(".show-hide").click(function() {
+  $(this).toggleClass("ms-icon icon-hide");
+  var input = $($(this).attr("toggle"));
```

Comment:
What do you think about making this smarter and search the input with the type password in current _field-item_? (and avoid using the reference)

Reply:
Oh, you cannot because after that you convert it into a text, but, well, there are two options:
* in that moment you could store a reference to the input.
* or, directly use the only input in this _field-group_, there should be only one.

Reply:
Yes, it is possible to implement the second item, store a reference to the entry to make the code smarter.